src/scripts/browserDeviceProfile.js

```diff
@@ -474,12 +484,15 @@ export default function (options) {
 
     if (supportsMp3VideoAudio) {
         videoAudioCodecs.push('mp3');
+    }
 
-        // PS4 fails to load HLS with mp3 audio
-        if (!browser.ps4) {
-            hlsInTsVideoAudioCodecs.push('mp3');
-        }
+    // Safari supports mp3 with HLS, but only in mpegts container, and the supportsMp3VideoAudio will return false.
+    if (!browser.ps4 && (browser.safari || supportsMp3VideoAudio)) {
```

Comment:
Probably matches the comment above better. 🤷
```suggestion
    if (browser.safari || (supportsMp3VideoAudio && !browser.ps4)) {
```